### PR TITLE
Increase rotation period

### PR DESCRIPTION
**Pull request by rrtti**
Increase the rotation period for candidates to show PoI FROM 3,3 days to 7 days.

### CODE REVIEW

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
I think this is "runtime worthy" since it will change some mechanics of Kusama

**Comment by kianenigma** [comment]
indeed, @rrtti perhaps share some links ahead of time with the society members to be aware and object if they wish to do so, thourgh governance.

**Review by kianenigma — APPROVED** [review]

**Comment by rrtti** [comment]
Changes have been announced on the Kappa Sigma Mu channel, will monitor response.

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -570,7 +570,7 @@ impl pallet_treasury::Config for Runtime {
 	type ProposalBondMinimum = ProposalBondMinimum;
 	type SpendPeriod = SpendPeriod;
 	type Burn = Burn;
-	type BurnDestination = Society;
+	type BurnDestination = ();
```
Comment (gavofyork) [inline runtime/kusama/src/lib.rs:573]:
```suggestion
	type BurnDestination = Society;
```

**Pull request merged**